snowshu/core/main.py

```diff
@@ -141,7 +141,6 @@ def create(replica_file: click.Path,  # noqa pylint: disable=too-many-arguments
     replica = ReplicaFactory()
     replica.load_config(replica_file, target_arch=target_arch)
 
-    replica.check_adapter_support(replica, incremental, '-i', 'incremental')
```

Comment:
why do we remove this?

Reply:
Honestly this was added to avoid error while running remote adapters with incremental flag, until we implement it.